src/core/PlayArea.ttslua

```diff
@@ -3,14 +3,17 @@
 ---------------------------------------------------------
 
 -- set true to enable debug logging
-local DEBUG = false
+local DEBUG = true
```

Comment:
Set to false

Reply:
I am far too used to runtime debug levels.  Takes me 10 minutes to figure out why my logs aren't showing anything, and I forget to turn it off every time